src/__tests__/trapAccess.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { KEYS, trapAccess } from '../trapAccess';
+
+describe('trapAccess', () => {
+  test('given no KEYS, should use own keys in ownKeys trap', () => {
+    const obj = trapAccess({ '404': null, '200': null });
+    expect(Reflect.ownKeys(obj)).toEqual(['200', '404']);
+  });
+
+  test('given no KEYS, should use KEYS in ownKeys trap', () => {
+    const obj = trapAccess({ '404': null, '200': null, [KEYS]: ['404', '200'] });
+    expect(Reflect.ownKeys(obj)).toEqual(['404', '200']);
+  });
```

Comment:
test is wrong

```suggestion
  test('given KEYS, should use KEYS in ownKeys trap', () => {
    const obj = trapAccess({ [KEYS]: ['404', '200'] });
    expect(Reflect.ownKeys(obj)).toEqual(['404', '200']);
  });
```

Reply:
Ha? It's valid. Not quite sure what you mean here.
Numeric keys are sorted in ascending order.

Reply:
```suggestion
  test('given KEYS, should use KEYS in ownKeys trap', () => {
    const obj = trapAccess({ '404': null, '200': null, [KEYS]: ['404', '200'] });
    expect(Reflect.ownKeys(obj)).toEqual(['404', '200']);
  });
```